lib/srv/db/common/auth.go

```diff
@@ -472,12 +511,16 @@ or "iam.serviceAccounts.getAccessToken" IAM permission.
 //
 // It is used to generate a one-time password when connecting to GCP MySQL
 // databases which don't support IAM authentication.
-func (a *dbAuth) GetCloudSQLPassword(ctx context.Context, sessionCtx *Session) (string, error) {
+func (a *dbAuth) GetCloudSQLPassword(ctx context.Context, database types.Database, databaseUser string, databaseName string) (string, error) {
```

Comment:
```suggestion
func (a *dbAuth) GetCloudSQLPassword(ctx context.Context, database types.Database, databaseUser string) (string, error) {
```
Or are we passing in the DB name for logging? In my opinion, the `Auth` shouldn't be concerned about `databaseName` if it doesn't need it. If we love to log them for context, we can initialize those in the logger when updating the logger for the session.

Reply:
Thanks for this. It was actually my intention to remove non-essential params, I just missed those because I've added logging logic for these.